packages/rich-text/README.md

```diff
@@ -14,169 +14,507 @@ _This package assumes that your code will run in an **ES2015+** environment. If
 
 ## Usage
 
-### create
+<!-- START TOKEN(Autogenerated API docs) -->
 
-```js
-create( {
-	?element: Element,
-	?text: string,
-	?html: string,
-	?range: Range,
-	?multilineTag: string,
-	?multilineWrapperTags: Array,
-} ): Object
-```
+### apply
 
-Create a RichText value from an `Element` tree (DOM), an HTML string or a plain text string, with optionally a `Range` object to set the selection. If called without any arguments, an empty value will be created. If `multilineTag` is provided, any content of direct children whose type matches `multilineTag` will be separated by a line separator.
+[src/index.js#L27-L27](src/index.js#L27-L27)
 
-### toHTMLString
+Create an `Element` tree from a Rich Text value and applies the difference to
+the `Element` tree contained by `current`. If a `multilineTag` is provided,
+text separated by two new lines will be wrapped in an `Element` of that type.
 
-```js
-toHTMLString( {
-	value: Object,
-	?multilineTag: string,
-	?multilineWrapperTags: Array,
-} ): string
-```
+**Parameters**
 
-Create an HTML string from a Rich Text value. If a `multilineTag` is provided, text separated by a line separator will be wrapped in it.
+-   **$1** `Object`: Named arguments.
+-   **$1.value** `Object`: Value to apply.
+-   **$1.current** `HTMLElement`: The live root node to apply the element tree to.
+-   **$1.multilineTag** `?string`: Multiline tag.
+-   **$1.multilineWrapperTags** `?Array`: Tags where lines can be found if nesting is possible.
+-   **$1.prepareEditableTree** `?Array`: Functions to prepare the editable tree.
```

Comment:
This attribute should be marked as unstable. It's was added for the annotations API, which is also unstable.

Reply:
Removing the att from the JSDoc will remove it from this as well. I just did that cfae06862fb708a367aa950dcb4d9573a637a220

This was something I documented because it wasn't (I didn't know it was supposed to be unstable/private).